CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# v0.0.44
+
+## Announcing Akri v0.0.44!
+Akri v0.0.44 is a pre-release of Akri.
+
+To find out more about Akri, check out our [README](https://github.com/deislabs/akri/blob/main/README.md) and start [contributing](https://github.com/deislabs/akri/blob/main/docs/contributing.md) today!
+
+## New Features
+The v0.0.44 release introduces a number of significant improvements!
+
+* Enable Akri for armv7
+* Create separate Helm charts for releases (akri) and merges (akri-dev)
+* Parameterize Helm for udev beyond simple video scenario
+* Expand udev discovery to include subsystems
+* Parameterize Helm for udev to allow security context
+* Remove requirement for agent to execute in privileged container
+
+View the [full change log](https://github.com/deislabs/akri/compare/v0.0.35...v0.0.44)
+
+## Breaking Changes
+N/A
+
+## Known Issues
+N/A
```

Comment:
Should we add supported K3s, K8s, MicroK8s versions here?

Reply:
Something like?

## Validated With
* Kubernetes: v1.19.4
* k3s: v1.18.9+k3s1
* MicroK8s: 1.18/stable

Reply:
maybe we need to include the known k3s version issue here as well?

Reply:
That would be good. It could link to an issue too.